execution/market.go

```diff
@@ -380,7 +380,7 @@ func NewMarket(
 
 	now := timeService.GetTimeNow()
 	liqEngine := liquidity.NewSnapshotEngine(
-		liquidityConfig, log, timeService, broker, tradableInstrument.RiskModel, pMonitor, asset, mkt.ID, stateVarEngine, mkt.TickSize(), priceFactor.Clone(), positionFactor)
+		liquidityConfig, log, timeService, broker, tradableInstrument.RiskModel, as, pMonitor, asset, mkt.ID, stateVarEngine, mkt.TickSize(), priceFactor.Clone(), positionFactor)
```

Comment:
As mentioned above: IMO it's cleaner to give `pMonitor` access to `as`, rather than to pass in `as` here only to call `as.InAuction()` when calling the `pMonitor`

Reply:
Addressed now I believe.